addons/point_of_sale/static/src/app/screens/ticket_screen/ticket_screen.js

```diff
@@ -593,7 +591,7 @@ export class TicketScreen extends Component {
      * @returns {Array} refundableDetails
      */
     _getRefundableDetails(partner, order) {
-        return Object.values(this.pos.lineToRefund).filter(
+        return Object.values(this.pos.linesToRedund).filter(
             (refund) =>
                 !this.pos.isProductQtyZero(refund.qty) &&
                 (partner ? refund.line.order_id.partner_id.id === partner.id : true) &&
```

Comment:
We cannot be sure that the `refund.line.order_id` has a partner as it can come from any orderline but the `partner` given to the method it the partner of the `selectedOrder`, so this might gives a traceback